plugins.json

```diff
@@ -1168,6 +1168,63 @@
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-dice-roller-v3.1.1.tar.gz",
+    "release_notes_url": "https://github.com/moussetc/mattermost-plugin-dice-roller/releases/tag/v3.1.1",
+    "labels": [
+      {
+        "name": "Community",
+        "description": "This plugin is maintained by the Open Source Community.",
+        "url": "https://mattermost.com/pl/default-community-plugins",
+        "color": ""
+      }
+    ],
+    "hosting": "",
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmBKEDIACgkQ0bVLR6XO/sSLgg/+IQPZ3IgiDfQ3DBxdzBhjIwV3srd5K8HqOD2BJKVIWs+8DFNSFwkDFR4/8zwU4qmoDw+Ni3IdO0A4Mgle28R8E7OKb5ax8PM4x+PvUyCR6FynuCP3ZLu8YAMCZEfva2/FTqAdJnWCuv/jwaYgPqPg7Uu1MYcDFryGrRfkPykDF0vUGLpTw7joG3UE4owaBYCKmY+xP4XMIGdsrIsGRvW8IrnjgZfzqO3jYT+5eCt3yw25LKPaHLQbfNp9FlWBTFwjXaXNmCR8zdzsBAMxKFGc5QiUMIfqMfTXiC9ULwo5fgxc2pEp/qAsTfE5iwDS07jN/t08rQdA2Y4TrG3LdAbNU/U/dFFSoFtQjtLmLv1CowQjNcgCTD1uDM7at+Nf9M/+9aR83bu72LeND8dQedNsBsoMlIQNgAM69pTW48qj6hiAnBBoRfJV7drkrGbUof++t9LQgUnvjnmw0sXGyPgdeJtpd8QcJjpWOA82tMgP8BlFJKUmaPLskTPTyTLCED6QAnwMAyLbm0Ml8xz0tAq8O4L4c9wkQG1Ndg5o1WkQt0zrDb6dFD++/DQtsO4DacNUYEUEe/ForKr1kYONQFNHDdyaLNWmiBotf/EEQif6LQuTSOoIAt7Ue7fwJkjbbe7H/lE2CMs39PhCXdiM/YQvANWo9X43+CXlsc6/uqo5hIg=",
+    "repo_name": "mattermost-plugin-dice-roller",
+    "manifest": {
+      "id": "com.github.moussetc.mattermost.plugin.diceroller",
+      "name": "Dice Roller ⚄",
+      "description": "Add a command for rolling dice ⚄",
+      "homepage_url": "https://github.com/moussetc/mattermost-plugin-dice-roller/",
+      "support_url": "https://github.com/moussetc/mattermost-plugin-dice-roller/issues",
+      "release_notes_url": "https://github.com/moussetc/mattermost-plugin-dice-roller/releases/tag/v3.1.1",
+      "icon_path": "assets/icon.svg",
+      "version": "3.1.1",
+      "min_server_version": "5.20.0",
+      "server": {
+        "executables": {
+          "linux-amd64": "server/dist/plugin-linux-amd64",
+          "darwin-amd64": "server/dist/plugin-darwin-amd64",
+          "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+        },
+        "executable": "server/dist/plugin-freebsd-amd64"
+      },
+      "settings_schema": {
+        "header": "",
+        "footer": "* To report an issue, make a suggestion or a contribution, [check the GitHub repository]i(https://github.com/moussetc/mattermost-plugin-dice-roller/)",
```

Comment:
Is the `i` in the middle intentional?

Reply:
You got eagle eyes! 

No, it's not intentionally. I only caught it after the plugin release was already done (but not pushed to the Marketplace, this is what this PR does). It's fixed upstream with 
 https://github.com/moussetc/mattermost-plugin-dice-roller/commit/a4597d56f873b09449c1e0a61b64fa3057786cf6 and the next release will contain the fix.